CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contribution Guidelines
+
+Ensure your pull request adheres to the following guidelines:
+
+- If you submit a project that is similar to an existing project in the list, argue how it's different or better.
+- Suggested packages should be tested and documented.
+- Make an individual pull request for each suggestion.
+- Use the following format: `[package](link) - Description.`
+- Additions should be added to the bottom of the relevant category.
+- Link to the GitHub repo.
+- Keep descriptions short and simple, but descriptive.
+- Start the description with a capital and end with a full stop/period.
+- Start the description with either a verb or a noun phrase.
+- Don't start the description with `A`, `An` or `The`.
+- Don't add `-s` suffix if the description starts with a verb.
+- Avoid `(for) Deno` in the description unless necessary. Stylize as `Deno` instead of `deno`.
```

Comment:
I realized that `deno fmt .` actually formats/ rewrites all the `Deno` word in `.md` files (can't recall what else file formats it affects) to `deno`.

@sindresorhus Can we check with @ry and fellow contributors on this? Which name should the community use?